### DISCUSSION TITLE
Query as anon user returns zero results if order ascending == true

### DISCUSSION THREAD

**Discussion by irwinb** [supabase/supabase#29009] [Questions] [answered]
I have a tiny table, with 4 columns. I am trying to sort by a column of type `text`.

* If `ascending` is `true`, it returns zero results.
* If `ascending` is `false`, it returns the correct results.
* If I run the query as a super_user, it returns the correct results.
* If I run the query on the Supabase dashboard SQL Editor, as an anon user, it returns the correct results.

I am using the NodeJS library, version `2.45.2`.

The code I am running:
```
await supabase
    .from('divisions')
    .select()
    .order('name', {
      ascending: true
    });
```

The table has the following policy:
```
alter policy "Enable read access for all users"
on "public"."divisions"
to public
using (
  true
);
```

Any help would be appreciated!

**Comment by GaryAustin1** [comment, marked as answer]
My guess is you ran it at some point in next.js and then changed the RLS or some such.  Next.js caches many url requests and so you are getting a cached url request.  Changing to ascending false changes the url.

**Reply by irwinb** [reply]
Yea, that explain it. Thanks!
